example/src/main/resources/templates/webeid-callback.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Thymeleaf namespace missing:

```suggestion
<html lang="en" xmlns:th="http://www.thymeleaf.org">
```